Userland/Libraries/LibWeb/CMakeLists.txt

```diff
@@ -281,11 +281,17 @@ function(libweb_js_wrapper class)
         )
     endforeach()
     add_custom_target(generate_${basename}Wrapper.h DEPENDS Bindings/${basename}Wrapper.h)
+    add_dependencies(all_generated generate_${basename}Wrapper.h)
     add_custom_target(generate_${basename}Wrapper.cpp DEPENDS Bindings/${basename}Wrapper.cpp)
+    add_dependencies(all_generated generate_${basename}Wrapper.cpp)
     add_custom_target(generate_${basename}Constructor.h DEPENDS Bindings/${basename}Constructor.h)
+    add_dependencies(all_generated generate_${basename}Constructor.h)
     add_custom_target(generate_${basename}Constructor.cpp DEPENDS Bindings/${basename}Constructor.cpp)
+    add_dependencies(all_generated generate_${basename}Constructor.cpp)
     add_custom_target(generate_${basename}Prototype.h DEPENDS Bindings/${basename}Prototype.h)
+    add_dependencies(all_generated generate_${basename}Prototype.h)
     add_custom_target(generate_${basename}Prototype.cpp DEPENDS Bindings/${basename}Prototype.cpp)
+    add_dependencies(all_generated generate_${basename}Prototype.cpp)
 endfunction()
 
```

Comment:
dunno if they're required but your scan for generated_\<foo\> seems to be missing generate_PropertyID.h, generate_PropertyID.cpp,  generate_ValueID.h generate_ValueID.cpp from the bottom of this file. Also the generated style sheet sources, but I don't think there's much code there just data.

Also missing the unicode data files. from unicode_data.cmake. Maybe that one would be better served with a 
```
set(GENERATED_SOURCES
      ${UNICODE_DATA_SOURCES}
)
```
in LibUnicode/CMakeLists.txt though.

Reply:
Fixed, thanks! I added you as `Co-authored-by:` since it was mostly your idea to begin with :)